storage/adapters/pgp_adapter.js

```diff
@@ -190,6 +183,24 @@ class PgpAdapter extends BaseAdapter {
 	parseQueryComponent(query, params) {
 		return this.pgp.as.format(query, params);
 	}
+
+	_getExecutionHandler(tx, expectedResultCount) {
+		const count = Number(expectedResultCount);
+		const context = tx || this.db;
```

Comment:
the function is called _getExecutionHandler but returns a context.[none|one|any] could the names of function and variables be matched? maybe change `context` to `dbHandler` or `_getExecutionHandler` to `_getDBContext` or similar?